{{cookiecutter.__src_folder_name}}/requirements-dev.in

```diff
@@ -1,15 +1,21 @@
 -r src/requirements.txt
-pytest
-coverage
-pytest-cov
-pytest-playwright
+
+# Cookiecutter Support
+cruft
```

Comment:
It feels a little weird for the developer-facing code to have cruft as a dependency, since I don't expect them to have to call cruft. A comment might be useful in the future.